compiler/qsc_qasm3/src/tests/assignment/slices.rs

```diff
@@ -0,0 +1,2 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
This file seems to be empty. Do we need it?

Reply:
An example of me thinking I need to test something but then couldn't get qiskit to generate it. Deleted.